libs/pika/execution/include/pika/execution/algorithms/ensure_started.hpp

```diff
@@ -10,36 +10,37 @@
 
 #if defined(PIKA_HAVE_STDEXEC)
 # include <pika/execution_base/stdexec_forward.hpp>
-#else
```

Comment:
Because
> - uses ensure_started and start_detached from pika since they have been removed from P2300 and will likely be removed from stdexec in the future; pika will keep them around longer until something async_scope-like is a viable replacement